data/json/bionics.json

```diff
@@ -64,34 +73,61 @@
-    "description": "The flesh on your head has been surgically replaced by alloy plating, protecting both your head and jaw regions.",
-    "occupied_bodyparts": [ [ "head", 5 ], [ "mouth", 1 ] ],
-    "bash_protec": [ [ "head", 3 ] ],
-    "cut_protec": [ [ "head", 3 ] ],
-    "bullet_protec": [ [ "head", 3 ] ],
-    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ]
+    "description": "The skin and flesh on your head has been surgically replaced by alloy plating, protecting both your head and jaw regions from physical and chemical injuries at the cost of flexibility.",
+    "occupied_bodyparts": [ [ "head", 6 ], [ "mouth", 2 ] ],
+    "bash_protec": [ [ "head", 7 ] ],
+    "cut_protec": [ [ "head", 6 ] ],
+    "bullet_protec": [ [ "head", 6 ] ],
+    "fire_protec": [ [ "head", 2 ] ],
+    "acid_protec": [ [ "head", 2 ] ],
+    "env_protec": [ [ "head", 2 ] ],
+    "encumbrance": [ [ "head", 10 ] ],
+    "wet_protection": [ { "part": "head", "ignored": 4 } ],
+    "covered_bodyparts": [ "head" ],
+    "no_bleed_chance": 0.25,
+    "no_bite_chance": 0.33,
+    "cant_uninstal_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
+    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTAL", "BIONIC_CANT_COMPRESS" ]
   },
   {
     "id": "bio_armor_legs",
     "type": "bionic",
     "name": { "str": "Alloy Plating - Legs" },
-    "description": "The flesh on your legs has been surgically replaced by alloy plating.  Provides passive protection and can be used in conjunction with bionic martial arts.",
-    "occupied_bodyparts": [ [ "leg_l", 6 ], [ "leg_r", 6 ] ],
-    "bash_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
-    "cut_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
-    "bullet_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
-    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ]
+    "description": "The skin and flesh on your legs has been surgically replaced by alloy plating.  Provides passive physical and chemical protection and can be used in conjunction with bionic martial arts at the cost of flexibility.",
+    "occupied_bodyparts": [ [ "leg_l", 7 ], [ "leg_r", 7 ] ],
+    "bash_protec": [ [ "leg_l", 7 ], [ "leg_r", 7 ] ],
+    "cut_protec": [ [ "leg_l", 5 ], [ "leg_r", 5 ] ],
+    "bullet_protec": [ [ "leg_l", 5 ], [ "leg_r", 5 ] ],
+    "fire_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
+    "acid_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
+    "env_protec": [ [ "leg_l", 2 ], [ "leg_r", 2 ] ],
+    "encumbrance": [ [ "leg_l", 4 ], [ "leg_r", 4 ] ],
+    "wet_protection": [ { "part": "leg_l", "ignored": 4 }, { "part": "leg_r", "ignored": 4 } ],
+    "covered_bodyparts": [ "leg_l", "leg_r" ],
+    "no_bleed_chance": 0.25,
+    "no_bite_chance": 0.33,
+    "cant_uninstal_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
+    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTAL", "BIONIC_CANT_COMPRESS" ]
```

Comment:
```suggestion
    "cant_uninstall_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTALL", "BIONIC_CANT_COMPRESS" ]
```